### PR TITLE
style: add gradient to main header; add blockquote styling; remove all caps

### PR DESCRIPTION
**Pull request by DSchau**
I'll release this as 1.0.0 so we can point to a specific version from npm

### CODE REVIEW

File: src/object-partners-revealjs-theme.scss [hunk 1]
```diff
@@ -25,6 +26,11 @@ pre code {
   color: #DDD;
 }
 
+.reveal section blockquote {
+	background: white;
```
Comment (DSchau) [inline src/object-partners-revealjs-theme.scss]: This isn't indented like this in my local copy 🙄

Comment (mcacek) [inline src/object-partners-revealjs-theme.scss]: tabs versus spaces?

Comment (DSchau) [reply src/object-partners-revealjs-theme.scss]: accurate!

**Review by mcacek — APPROVED** [review]

**Comment by mcacek** [comment]
👍

**Pull request merged**